module/documents/actor/actor.mjs

```diff
@@ -260,11 +263,13 @@ export default class Actor5e extends Actor {
   _prepareBaseSkills() {
     if ( !("skills" in this.system) ) return;
     const skills = {};
+    const skillField = dnd5e.utils.getSchemaField(this.system, "skills.*");
     for ( const [key, skill] of Object.entries(CONFIG.DND5E.skills) ) {
       skills[key] = this.system.skills[key];
       if ( !skills[key] ) {
-        skills[key] = foundry.utils.deepClone(game.system.template.Actor.templates.creature.skills.acr);
-        skills[key].ability = skill.ability;
+        const initial = skillField.getInitialValue();
+        initial.ability = skill.ability;
+        skills[key] = skillField.initialize(initial, this.system);
```

Comment:
I would have expected this to be handled by `CreatureTemplate._initialSkillValue`. Seems to be related to the `template.json` `getInitialValue` I mentioned earlier.